File: 2-13认证杯/08-20年挑战赛优秀论文汇总/2013第六届一阶段优秀论文/第一阶段/b/特等奖/1023队/code/framework/matlab_skeleton/DescriptorExtractor.m
function DescriptorExtractor(filename, resultfile)

% reading the audio
[audio, samplerate] = wavread(filename);

% the processing goes here...





% writing results to file
f = fopen(resultfile, 'w');

% replace this line with your code...
fprintf(f, '%d %d %d...\n', 12, 34, 56);

% we are done, closing file
fclose(f);

File: 2-13认证杯/08-20年挑战赛优秀论文汇总/2013第六届一阶段优秀论文/第一阶段/b/特等奖/1023队/code/framework/python_skeleton/DescriptorExtractor.py
#!/usr/bin/python

import wave
import sys
from numarray import *

if len(sys.argv) != 3:
    print 'usage: ', sys.argv(0), ' filename resultfile'
    print 'where filename is the name of the audio file to analyse'
    sys.exit(1)

# get arguments from command line
filename = sys.argv[1]
resultfile = sys.argv[2]

# read the audio
audio_file = wave.open(filename, 'r')
n = audio_file.getnframes()
audio = array(audio_file.readframes(n), Int16).astype(Float32) / 32768.0

# the processing goes here...






# writing results to file
out = open(resultfile, 'w')

# replace this line with your code...
out.write('12 34 56\n')

# we are done, closing file
out.close()
